test/helpers/deployer-duelist-king.ts

```diff
@@ -85,6 +95,7 @@ export default async function init(context: {
           registryRecords.domain.duelistKing,
           registryRecords.domain.duelistKing,
           registryRecords.domain.duelistKing,
+          registryRecords.domain.duelistKing,
```

Comment:
You might need to deploy the staking separately.

Reply:
I will implement the code that deploying registry contract and this staking contract only for testing